packages/integration-tests/playwright.config.ts

```diff
@@ -27,7 +27,7 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 1 : undefined,
 
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
+  reporter: [['list'], ['html']],
```

Comment:
In GitLab CI, we were just getting the [`dot` reporter](https://playwright.dev/docs/test-reporters#dot-reporter) which made failures difficult to debug.

This change forces the default reporter (`list`) in addition to `html`.